skyvern-frontend/src/routes/tasks/detail/TaskDetails.tsx

```diff
@@ -267,6 +276,7 @@ function TaskDetails() {
         <>
           {extractedInformation}
           {failureReason}
+          {webhookFailureReason}
```

Comment:
if failure reason and webhook failure reason are both there, how does it look like?